packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -159,6 +160,9 @@ $side-padding: 1.2em;
                 }
             }
         }
+        .data-list__desc--wrapper {
+            justify-content: center;
+        }
```

Comment:
It is better to use SCSS nesting capabilities instead of defining separate classes

Reply:
That class was predefined in `data-list-row.jsx`. But I can break it down like this:
```
.data-list {
            &__desc--wrapper {
                justify-content: center;
            }
        }
}
```

How about that?